spec/numberformat.html

```diff
@@ -30,7 +30,8 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
           1. Set _intlObj_.[[MaximumSignificantDigits]] to _mxsd_.
         1. Else if _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, then
           1. Set _intlObj_.[[RoundingType]] to ~fractionDigits~.
-          1. Let _mnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, _mnfdDefault_).
+          1. Let _mnfdActualDefault_ be ? min( _mnfdDefault_, DefaultNumberOption(_mxfd_, 0, 20, 20) ).
```

Comment:
This will lead to calling `ToNumber` twice on `mxfd`, which is observable if `mxfd` is an object. Do we care about this?

Two nits:
1. Can you swap the arguments to `min` so we have `min(DNO(...), mnfdDefault)`? That way the arguments are consistent with the `max` call further below.
1. `DefaultNumberOption(_mxfd_, 0, 20, mnfdDefault)` instead of `DefaultNumberOption(_mxfd_, 0, 20, 20)` might be slightly nicer, because than it's directly visible what's the default value for `mnfdActualDefault`.

Reply:
I do care about invoking ToNumber on the same value more than once, which should be avoided. I'm also uncomfortable with embedding a potentially-abrupt abstract operation as the parameter of another.

What about something a little more explict/straightforward?
```suggestion
          1. Set _specifiedMnfd_ to ? DefaultNumberOption(_mnfd_, 0, 20, *undefined*).
          1. Set _specifiedMxfd_ to ? DefaultNumberOption(_mxfd_, 0, 20, *undefined*).
          1. If _specifiedMxfd_ is not *undefined*, set _mnfdDefault_ to min(_mnfdDefault_, _specifiedMxfd_).
          1. Set _mnfd_ to ! DefaultNumberOption(_specifiedMnfd_, 0, 20, _mnfdDefault_).
          1. Set _mxfd_ to ! DefaultNumberOption(_specifiedMxfd_, 0, 20, max(_mxfdDefault_, _mnfd_)).
          1. If _mnfd_ is greater than _mxfd_, throw a *RangeError* exception.
```
(even more so if we replace the second set of DefaultNumberOption with explicit remapping of undefined values)

Reply:
@gibson042 thanks for the suggestion. Changing my implementation to this produced the expected results, so this should work fine. I'll change the PR to this.